.github/workflows/ci.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       postgres:
-        image: postgres:11
+        image: postgres:13
```

Comment:
Any objections to this? This is the version we're currently using for our local db setup: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/scripts/local_db_setup.sh#L21

Reply:
+1, let's go with the change.

Reply:
+1